src/components/Cart.js

```diff
@@ -13,6 +15,7 @@ export const Cart = () => {
     <HeaderNavBar/>
```

Comment:
Why are you adding HeaderNavbar here? Isn't it a common component? This can be added as below

```JSX
   <HeaderNavbar />
   <Routes>
       /*Define your routes here*/    
   </Routes>
   <Footer />
```

In this way the component matching the route in address bar will be displayed along with Header and Footer

Reply:
I will raise an issue to fix this. Grid structure of the layout of the pages is tied to the header/footer and its causing issues. Will refactor by raising an issue.